src/configuration/app-containers.md

```diff
@@ -76,466 +73,18 @@ hooks:
-environment variable: `$PLATFORM_BRANCH` which you should put in an `if/then` statement.
-
-Your `.platform.app.yaml` file should include:
-
-```yaml
-hooks:
-  deploy: |
-    if [ "$PLATFORM_BRANCH" = master ]; then
-      # Use Drush to disable the Devel module on the Master environment.
-      drush dis devel -y
-    else
-      # Use Drush to enable the Devel module on other environments.
-      drush en devel -y
-      # Sanitize your database and get rid of sensitive information from Master environment.
-      drush -y sql-sanitize --sanitize-email=user_%uid@example.com --sanitize-password=custompassword
-    fi
-    drush -y updatedb
-```
-
-## Crons and Cron jobs
-
-Cron jobs allow you to run scheduled tasks at specified times or intervals. The `crons` object describes these tasks and the schedule when they are triggered.  Each item in the list is a unique name identifying a separate cron job.
-
-It has a few subkeys listed below:
-
--   **spec**: The [cron specification](https://en.wikipedia.org/wiki/Cron#CRON_expression). For example: `*/20 * * * *` to run every 20 minutes.
--   **cmd**: The command that is executed, for example
-    `cd public ; drush core-cron`
-
-The minimum interval between cron runs is 5 minutes, even if specified as less.
-
-```yaml
-crons:
-    # Run Drupal's cron tasks every 20 minutes.
-    drupal:
-        spec: '*/20 * * * *'
-        cmd: 'cd web ; drush core-cron'
-    # But also run pending queue tasks every 5 minutes.
-    drush-queue:
-        spec: '*/5 * * * *'
-        cmd: 'cd web ; drush queue-run aggregator_feeds'
-```
-
-Cron runs are executed using the dash shell, not the bash shell used by normal SSH logins. In most cases that makes no difference but may impact some more involved cron scripts.
-
-A variable delay is added to each cron job in each project in order to prevent host overloading at midnight every night when every project runs their nightlies.  Your crons will *not* run exactly at the time that you specify, but will be delayed by 0-300 seconds.
-
-All server times are in UTC, and hence so are the cron schedules.
-
-
-## Runtime
-
-The `.platform.app.yaml` file also supports a `runtime` key that allows selected customizations to the language runtime. As those possibilities vary by language, please see the appropriate language documentation.
-
-* [PHP](/languages/php.md)
+The `.platform.app.yaml` file is extremely flexible.  Depending on your needs it could be less than 10 lines long or over 100.  The only required keys are `name`, `type`, `disk`, and a minimal `web` block.  All others are optional.
 
-## Top level document roots
+The basic properties are described on their own pages.
 
-Platform.sh requires that the document root not be at the root of the project.  It is important for security that
-private file mounts are not web-accessible.
+* [`name`](/configuration/app/name.php) (required) - Sets the unique name of the application container.
```

Comment:
All the links are broken in this list. I guess they should end with `.md`, not `.php`